frontend/src/components/Main/AnalysisForm/AnalysisForm.js

```diff
@@ -75,6 +76,16 @@ class AnalysisForm extends Component {
                 {mainStore.errorMessage ? (
                     <div className="alert alert-danger">{mainStore.errorMessage}</div>
                 ) : null}
+
+                <div>
+                    <p>Steps Required: </p>
+                    <p style={{float: "left"}}>1. Select a Model</p>
```

Comment:
Change the div/phrasing so it looks like this:

---


**Steps required to save and/or execute:**

- [x] At least one model is selected
- [ ] At least one dataset is selected
- [x] At least one option set is created

Reply:
completed